### PR TITLE
update paragon-ntfs to version 15.0.911_000

### PR DESCRIPTION
**Pull request by qinxin0720**
After making all changes to the cask:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] `sha256` changed but `version` stayed the same ([what is this?][version-checksum]).  
      I’m providing public confirmation below.

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not already refused in [closed issues].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/paragon-ntfs.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'paragon-ntfs' do
-  version '15'
-  sha256 :no_check # required as upstream package is updated in-place
+  version '15.0.911_000'
+  sha256 'dab9778908cbecc1b136e6e05fcf3ec350896c82b1d43c0a9bbaefdfde66df67'
 
-  url "http://dl.paragon-software.com/demo/ntfsmac#{version}_trial.dmg"
+  url "http://dl.paragon-software.com/esd/Paragon-1091-PEU_MacInstallUnlock_#{version}.dmg"
```
Comment (commitay) [inline Casks/paragon-ntfs.rb]: Where did you get this URL from?

**Comment by qinxin0720** [comment]
The paragon official website

发自我的 iPhone

在 2017年8月25日，下午2:43，commitay <notifications@github.com<mailto:notifications@github.com>> 写道：


@commitay commented on this pull request.

________________________________

In Casks/paragon-ntfs.rb<https://github.com/caskroom/homebrew-cask/pull/37979#discussion_r135192670>:

>
-  url "http://dl.paragon-software.com/demo/ntfsmac#{version}_trial.dmg"
+  url "http://dl.paragon-software.com/esd/Paragon-1091-PEU_MacInstallUnlock_#{version}.dmg"


Where did you get this URL from?

**Comment by commitay** [comment]
> The paragon official website

Which page?

**Comment by commitay** [comment]
> MyParagon account page

We can't use a download link from behind a login because when the version changes we can't access the new URL. Thank you anyway for the PR!

**Pull request closed without merging**